modules/yieldmoBidAdapter.js

```diff
@@ -26,7 +26,7 @@ export const spec = {
    * @param {BidRequest[]} bidRequests A non-empty list of bid requests which should be sent to the Server.
    * @return ServerRequest Info describing the request to the server.
    */
-  buildRequests: function(bidRequests) {
+  buildRequests: function(bidRequests, bidderRequest) {
     let serverRequest = {
       p: [],
       page_url: utils.getTopWindowUrl(),
```

Comment:
This function will be deprecated in prebid 3.0. More details about the deprecations [here](https://github.com/prebid/Prebid.js/issues/3072). Instead of using those deprecated functions, you can read the info from the bidRequest object. Find more details [here](http://prebid.org/dev-docs/bidder-adaptor.html#referrers).

Reply:
@Fawke thank you for pointing that out. I have updated the 2 functions that will be deprecated.